config.yaml

```diff
@@ -91,6 +91,8 @@ clouds:
     # Accounts to share newly created AMIs with
     grant_users:
       - "013116697141"
```

Comment:
Aside: we should probably add a comment to this one too. IIRC it's the Fedora community account, so that kola can use it for tests.

Reply:
Let me know if this is good.